plugins/opds.koplugin/opdspse.lua

```diff
@@ -27,10 +31,10 @@ function OPDSPSE:getLastPage(item, remote_url)
     -- Do an HTTP POST to get the Bearer Token for authentication of the /api/Reader/get-progress endpoint
     local auth_parsed = url.parse(auth_url)
     local auth_data = {}
-    local code, headers, status
+    local auth_code
     if auth_parsed.scheme == "http" or auth_parsed.scheme == "https" then
         socketutil:set_timeout(socketutil.FILE_BLOCK_TIMEOUT, socketutil.FILE_TOTAL_TIMEOUT)
-        code, headers, status = socket.skip(1, http.request {

```

Comment:
Keep in mind the triplet is necessary for proper error logging (c.f., #9523).

Reply:
If I leave it in, though, the build test generates a warning that `headers, status` are never accessed. Is there a way to leave them in but not have them throw that warning?

Reply:
While there are numerous ways, given what @NiLuJe wrote I think the more obvious answer is simply to use it:
```lua
        logger.dbg("WebDavApi:listFolder: Request failed:", status or code)
        logger.dbg("WebDavApi:listFolder: Response headers:", headers)
        logger.dbg("WebDavApi:listFolder: Response body:", table.concat(sink))
```

Reply:
Oh. 🤦 . I'll do that.